src/components/forms/power-vs/PowerVsVolumeForm.js

```diff
@@ -199,6 +207,7 @@ class PowerVsVolumeForm extends React.Component {
         />
         <IcseFormGroup>
           <IcseToggle
+            key={(this.state.pi_volume_pool || this.state.pi_storage_pool) + this.state.pi_replication_enabled} // force rerender when pool changes
```

Comment:
The key was added so that changes to the storage pool cause the toggle to automatically update based on the pool.